### ISSUE TITLE
Reorder calibration screens

### ISSUE THREAD

**Issue by thormj** [MaslowCNC/GroundControl#593]
I think it would be convenient to reorder the calibration screens like this:
1. Intro
2. Z-Axis calibration (since you really should adjust this when it's not on the frame)
3. Zero XY Motors
4. Measure XY distance
5. Pick sled type (we can store the value and proceed on 2 paths here)
6q. Measure Sled's Width (skip this if Triangle)
7.  Measure the top of the work area
8.  Extend Chains
9 Test pattern cut

**Comment by BarbourSmith** [comment]
I agree, with the one exception that I like to do my z-axis adjustment on the frame so that I can lower the bit until it just touches the wood. If others do it differently I could be convinced to try something different

**Comment by servant74** [comment]
One thought, it would be nice to be able to do something like put a small
sheet of aluminum foil below the router and lower till the bit touches as a
'touch off'.  Then either remove the foil or just 'route through it'.  Or
to use a more 'normal' touch off sensor, with a known height.

... Enhancement thoughts never cease! <grin>

**Comment by BarbourSmith** [comment]
The good news is that is already supported! 😀

**Comment by thormj** [comment]
The other "improvement" to this is that I would like to be able to enter pulses per mm for z height so we can support anything you can attach the z-motor to.  So I would zero-z.  Retract 10mm (or 5mm) and enter the height measured by my tire-depth gauge, and presto.

@BarbourSmith - what do you hook the wire to for touch-z?

**Comment by BarbourSmith** [comment]
I like the idea, but I think it might confuse people when 99% of people would have the default setting. I'm in support as long as we make it clear that most people don't need to do it.

I could guess, but there is already a forum thread where I did that and someone corrected me 😀 

https://forums.maslowcnc.com/t/automatic-tool-height/1839

**Comment by davidelang** [comment]
On Mon, 29 Jan 2018, Jack in TN wrote:

> One thought, it would be nice to be able to do something like put a small
> sheet of aluminum foil below the router and lower till the bit touches as a
> 'touch off'.  Then either remove the foil or just 'route through it'.  Or
> to use a more 'normal' touch off sensor, with a known height.

This functionality is already available in the firmware
